default/stringtables/en.txt

```diff
@@ -6283,6 +6295,21 @@ EFFECT_BIOWEAPON
 EFFECT_BIOWEAPON_LABEL
 Bioweapon Attack
 
+EFFECT_PLANET_TARGETED
+Our planet % planet is being targeted by a planet destroyer cannon!
```

Comment:
this looks malformed...

Reply:
Style is 
Planetname: stuff happens 

And as many pedia links as you can, and ideally a link to the attacking ship. 

Also remember to put the sitrep into the priority order list so it appears near the top.

Reply:
... ?? what? I have no idea what this comment is referring to.

Reply:
Sorry, was on phone.
default/customizations/common_user_customizations.txt

I don't know why it's in that file, it used to be in the stringtable, it's right that it's not there but it's not really a customisation and it's important that it's updated for new effects.

Reply:
Done, thanks for pointing out.

Reply:
I still have no idea what you're talking about.

The point is that the stringtable entry this comment thread is pointing to looks malformed. The lone % doesn't make sense.

Reply:
Ah, I was commenting on the actual text which was badly formatted and not per current style, both have subsequently been fixed.